udev-rules/CMakeLists.txt

```diff
@@ -7,8 +7,9 @@ project(
 
 checkdebianchangelogversion(${PROJECT_NAME})
 
+option(CONDA_UDEV_RULES "Install UDEV RULES into conda package" OFF)
 set(UDEV_RULES_INSTALL_PATH
-    "/etc/udev/rules.d"
+    "$<IF:$<BOOL:${CONDA_UDEV_RULES}>,${CMAKE_INSTALL_PREFIX}/etc/udev/rules.d,/etc/udev/rules.d>"
```

Comment:
No need for conditionals, UDEV_RULES_INSTALL_PATH is cached variable, you can override it from command line "cmake -DUDEV_RULES_INSTALL_PATH="somepath"